domain/apiresponses/responses.go

```diff
@@ -71,7 +71,8 @@ type BindingResponse struct {
 
 type GetBindingResponse struct {
 	BindingResponse
-	Parameters any `json:"parameters,omitempty"`
+	Parameters any               `json:"parameters,omitempty"`
+	Endpoints  []domain.Endpoint `json:"endpoints,omitempty"`
```

Comment:
`endpoints` is a property defined in both the create binding response and also in the the get binding response so it should be declared in `BindingResponse` struct.

Reply:
I have moved this as requested in commit [cdd667b](https://github.com/pivotal-cf/brokerapi/pull/321/commits/cdd667b6e426e101ef63a463f02bfa0c35bce88b).